functions/Backup-DbaDatabase.ps1

```diff
@@ -416,6 +428,16 @@ function Backup-DbaDatabase {
                     else {
                         $device.DeviceType = "File"
                     }
+                    
+                    if ($WithFormat) {
+                        Write-Message -Message "WithFormat specified. Ensuring Initialize and SkipTapeHeader are set to true." -Level Verbose
```

Comment:
more relevant for a debug level rather than verbose ....

Reply:
it seems important tho - cuz it changes people's settings even though they chose not to. I felt like it was either a warning / prompt or verbose and decided on verbose. You dont think it's important?

Reply:
I think we should log by default to a MINIMUM since we don't have any "knob" to tune logging verbosity. if it's so important because it changes people's settings it should warn and stop. if it's a requirement (as it seems written in code), either you go ahead and "do the right thing" 'cause not doing it results always in an error (but doing so, there's no reason to provide more than one additional switch) or you relinquish responsibilities stopping if the user isn't specifically asking for the combination that do NOT result in an error.

Reply:
we do have knobs via `Set-DbaConfig` 👯  (tho i dont recall which one but fred showed me before)

Reply:
yeah but not on the function itself, so logging anything to verbose and expecting users to "ingest" loads of not-so-important-messages via a single -Verbose switch isn't feasible

Reply:
i tried to change it to debug but my brain insisted it was fasho verbose 🤪